pkg/ddl/notifier/store.go

```diff
@@ -82,15 +82,26 @@ func (t *tableStore) UpdateProcessed(
 	return err
 }
 
-//revive:disable
-
-func (t *tableStore) Delete(ctx context.Context, se *sess.Session, ddlJobID int64, multiSchemaChangeID int) error {
-	//TODO implement me
-	panic("implement me")
+func (t *tableStore) DeleteAndCommit(
+	ctx context.Context,
+	se *sess.Session,
+	ddlJobID int64,
+	multiSchemaChangeID int,
+) error {
+	if err := se.Begin(ctx); err != nil {
+		return errors.Trace(err)
+	}
+	sql := fmt.Sprintf(`
+		DELETE FROM %s.%s
+		WHERE ddl_job_id = %d AND multi_schema_change_seq = %d`,
+		t.db, t.table,
+		ddlJobID, multiSchemaChangeID)
+	if _, err := se.Execute(ctx, sql, "ddl_notifier"); err != nil {
+		return errors.Trace(err)
+	}
+	return errors.Trace(se.Commit(ctx))
```

Comment:
Should we call `Rollback()` when it fails like we did in `processEventForHandler()`?